src/app/pulse/charts/aggregation-selector.component.ts

```diff
@@ -48,15 +48,18 @@ export class AggregationSelectorComponent implements OnChanges {
   @Input() public shiftAmount: number;
   selectedAggregations: Array<string>;
 
-  constructor() {
+  constructor(private storage: LocalStorageService) {
     this.emitShiftChange = debounce(this.emitShiftChange, 300);
   }
 
   public ngOnChanges(changes: SimpleChanges) {
     if ('permittedIntervals' in changes) {
       if (this.permittedIntervals) {
         setTimeout(() => {
-          this.selectedShift = this.permittedIntervals.shifts[0];
+          const storedShift = this.storage.read(PulseParameters.Shift);
```

Comment:
Please use the same method for fetching params here and https://github.com/bwsw/cloudstack-ui/pull/399/files#diff-f10a458d34561252887f3f5b3dc30d6dR246